examples/shared_infra/dataplane-shared.yaml

```diff
@@ -1,11 +1,11 @@
 dataplane:
   cloudType: aws
   cloudRegion: us-east-1
-  saasType: shared
+  saasType: private
   cloudAuth:
```

Comment:
@pkbhowmick lets create another directory for private. 
like this ```examples/private_infa/dataplane-private.yaml```

Reply:
updated